frame/asset-conversion/src/lib.rs

```diff
@@ -1086,7 +1090,7 @@ pub mod pallet {
 		/// Returns the next pool asset id for benchmark purposes only.
 		#[cfg(any(test, feature = "runtime-benchmarks"))]
 		pub fn get_next_pool_asset_id() -> T::PoolAssetId {
-			NextPoolAssetId::<T>::get().unwrap_or(T::PoolAssetId::initial_value())
+			NextPoolAssetId::<T>::get().or(T::PoolAssetId::initial_value()).unwrap()
```

Comment:
Is there really a difference here? If not, I actually prefer `unwrap_or` just to reduce the chances of panicking.

Reply:
Since the `T::PoolAssetId::initial_value()` returns an Option, unwrap_or() won't work anymore here

Reply:
I think there is a difference. `initial_value` returns an `Option`  and the unwrap_or expects an `T::PoolAssetId` as input. And as it is for testing its fine if it pannicks right?

Reply:
> Since the `T::PoolAssetId::initial_value()` returns an Option now, `unwrap_or()` won't work here anymore

Oh right, then in fact the code now as it is is wrong. It should also handle the case where `initial_value` returns `None`, otherwise we would panic here.

Reply:
Actually, nevermind, this code is only usable in test or runtime benchmarks. However, an `expect` is much more preferred here rather than an unwrap, as it gives a better error message.